docs/deploy.md

```diff
@@ -95,6 +95,17 @@ Follow the installation guide at the [Readme page](https://github.com/Microsoft/
 
 6. For more options over the deployment, follow the [Development guide](https://github.com/Microsoft/presidio/blob/master/docs/development.md)
 
+### Expose Presidio publicly
+We recommend exposing Presidio using a DNS name.
+The recommended option is using nginx-ingree with TLS:
+1) Deploy presidio with [ingress set to 'enable'](https://github.com/microsoft/presidio/blob/master/charts/presidio/values.yaml#L35).
```

Comment:
maybe note that you need ingress before. (perhaps  add reference to docs on aks for that)